CMakeLists.txt

```diff
@@ -4,8 +4,13 @@ project(itlab_2023)
 
 enable_testing()
 
+include_directories("include")
+
 add_subdirectory(3rdparty)
 add_subdirectory(app)
 add_subdirectory(include)
 add_subdirectory(src)
 add_subdirectory(test)
+
```

Comment:
Move benchmarking related stuff to `src/perf` or something similar

Reply:
Moved